### DISCUSSION TITLE
Remove "NVApi couldn't initialize or has no sensors" on laptops

### DISCUSSION THREAD

**Discussion by Kagetora120903** [Rem0o/FanControl.Releases#2683] [Ideas]
As the title says, could you please please PLEASE remove this warning? I don't want to know for the 10 billionth time that my laptop doesn't have that sensor or whatever causes it!

![Screenshot 2024-08-24 112941](https://github.com/user-attachments/assets/3393c5ae-06fd-4f9f-bcaa-a1af9eb9d6cc)

**Comment by Rem0o** [comment]
You can disable the sensor source. 
![image](https://github.com/user-attachments/assets/565613a7-4cff-46f6-8ccd-4b3f8702bdb4)
